pynamodb/migration.py

```diff
@@ -119,10 +129,17 @@ def migrate_boolean_attributes(model_class,
                     if code == 'ConditionalCheckFailedException':
                         log.warn('conditional update failed (concurrent writes?) for object: %s (you will need to re-run migration)', item)
                         num_update_failures += 1
+                    elif code == 'ProvisionedThroughputExceededException':
```

Comment:
it looks like the we have a group of code to check. let's take advantage of enum to avoid typo and the ease of reuse if ever.

Reply:
I am not sure if we have control over this (it seems like it's related to boto library/pynamo interaction, and/or code wide change) I think it's out of scope of this PR

Reply:
well, since this is tmp migration code and will NOT be in prod anyway, i will let it be, so we can save some time and get this out asap